explainer.md

```diff
@@ -605,7 +605,7 @@ enum XREnvironmentBlendMode {
   "alpha-blend",
 };
 
-dictionary XRRenderStateOptions {
+dictionary XRRenderStateInit {
   double depthNear;
   double depthFar;
```

Comment:
Even then, the API could accept `depthNear`/`depthFar` at the end of the frame, rather than during the previous frame.

Reply:
Are you asking for a change or this is a +1 or something else?

Reply:
Just setting context as we revisit `XRRenderState*` here that we may be able to do away with `depthNear`/`depthFar`, if that simplifies the reshuffle here.  Some platforms had previously needed the near/far planes to be set on the previous frame, but this is no longer the case for Windows MR.

Reply:
In the latest iteration, I noticed that all of the sample code only updated the `baseLayer` of the render state and never the `depthNear/depthFar`.  Was this done for sample code brevity or because you're proposing we remove depthNear and depthFar.  If it is the latter, then you need to remove the two variables from the explanatory text and IDL as well.  If the former, then you're good.

Reply:
Just for code brevity.